tools/cli/internal/openapi/filter/code_sample.go

```diff
@@ -121,6 +125,41 @@ func newAtlasCliCodeSamplesForOperation(op *openapi3.Operation) codeSample {
 	}
 }
 
+func (f *CodeSampleFilter) newGoSdkCodeSamplesForOperation(op *openapi3.Operation, opMethod string) codeSample {
+	version := strings.ReplaceAll(apiVersion(f.metadata.targetVersion), "-", "") + "001"
```

Comment:
This field represents the SDK version. The value is formatted as `YYYY-MM-DD-001`, where `001` indicates the first SDK release for that API version. The last three digits are incremented for subsequent releases, but `001` will always be available as the initial version for each API version.

Reply:
No, neither of those options apply. I generate the SDK version by combining the API version in the format `YYYY-DD.MM `with `001` as the suffix, which represents the first release for any new API version and is always present. Subsequent releases for the same API version will follow the format `YYYY-DD.MM-002`, `YYYY-DD.MM-003`, and so on, but the example will always use the initial `001` release.
If I merge this, I will then create a ticket to improve this to check for the release in the sdk repo.